requirements/all.txt

```diff
@@ -185,12 +181,10 @@ tomli==2.0.1
     #   autoflake
     #   black
     #   coverage
-    #   mypy
     #   pytest
 typing-extensions==4.5.0
     # via
     #   dirty-equals
-    #   mypy
```

Comment:
is mypy still in the testing dependencies?

Reply:
Seems it should probably be in the all since it is necessary to run some of the tests